src/components/tiles/Tile.tsx

```diff
@@ -65,7 +65,7 @@ export class TileComponent extends React.PureComponent<Props> {
       )
     }
     return (
-      <TouchableWithoutFeedback onPress={this.handlePress} disabled={type === 'static'}>
+      <TouchableWithoutFeedback accessible={false} onPress={this.handlePress} disabled={type === 'static'}>
```

Comment:
What children were made accessible here?

Reply:
All the content within the Tile such as the title